### PR TITLE
luci-app-sqm: convert to client side implementation

### PR DESCRIPTION
**Pull request by Ansuel**
Convert sqm app to client side implementation. Only a small rework in the logic to get qos scripts help text. The rest of the script is a porting 1:1.

Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>

### CODE REVIEW

**Comment by hnyman** [comment]
cc also @tohojo @moeller0

File: applications/luci-app-sqm/root/usr/share/luci/menu.d/luci-app-sqm.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+	"admin/network/sqm2": {
```
Comment (tohojo) [inline applications/luci-app-sqm/root/usr/share/luci/menu.d/luci-app-sqm.json:2]: nit: why sqm2?

**Comment by tohojo** [comment]
Thanks for doing this! Need to spin up an up-to-date openwrt to test it, will try to do so one of the coming days...

File: applications/luci-app-sqm/root/usr/share/luci/menu.d/luci-app-sqm.json (hunk 1 above)
Comment (Ansuel) [reply applications/luci-app-sqm/root/usr/share/luci/menu.d/luci-app-sqm.json:2]: (while I was testing this I renamed it to sqm2 to have both the old and the new implementation) will fix.
Any other comment?

Comment (tohojo) [reply applications/luci-app-sqm/root/usr/share/luci/menu.d/luci-app-sqm.json]: Thanks! And no, not yet, will need to take it for a spin first...

**Review by tohojo — CHANGES_REQUESTED** [review]
Right, took this for a spin, looks good!

You'll need to also update the Makefile, though: I would actually suggest just dropping the `PKG_VERSION` and `PKG_RELEASE` and just switching to the automatic date-based versioning used for the other Luci apps. Also, I believe this means you can drop the `luci-compat` dependency.

**Comment by Ansuel** [comment]
@hnyman think this can be merged

**Review by tohojo — APPROVED** [review]
LGTM

**Pull request merged**